native/main.ts

```diff
@@ -11,7 +11,7 @@ import { execFile, ChildProcess } from "child_process";
 import { RendererMessage, MainMessage, MainMessageKind } from "./ipc-types";
 
 const isDev = process.env.NODE_ENV === "development";
-const proxyPath = path.join(__dirname, "../../proxy");
+const proxyPath = path.join(__dirname, "../proxy/target/debug/api");
```

Comment:
This is only the location in development. What's the reason to change it?

Reply:
This was a development artifact. Fixed in https://github.com/radicle-dev/radicle-upstream/pull/1216/commits/50a29d915751601d44386d6a3972728de467e554